samples/snippets/cpp/VS_Snippets_Misc/NVC_MFC_VisualStudioDemo/cpp/VisualStudioDemo.rc

```diff
@@ -185,7 +185,7 @@ FONT 8, "MS Sans Serif", 0, 0, 0x1
 BEGIN
     ICON            IDR_MAINFRAME,IDC_STATIC,11,17,20,20
     LTEXT           "VisualStudioDemo Application",IDC_STATIC,40,10,119,8
-    LTEXT           "� 2007 - 2008 Microsoft Corporation.  All rights reserved.",
+    LTEXT           "© 2007 - 2008 Microsoft Corporation.  All rights reserved.",
```

Comment:
Looks like there are still unwanted changes in resource files. You claim in commit message that you fixed the encoding of rc files, but do you realize that resource compiler (even as of VS2019!) does not support UTF-8? Native resource files should only be in ANSI or UTF-16.

Reply:
Thanks, I did try converting to UTF-16, but it was also garbled. I'll see if `.gitattributes` can ensure it stays correct, but I'm guessing it might be broken today

Reply:
I think .gitattributes only configure encoding for purposes of detecting line endings and calculating diffs in GUI. What's more important is that the actual text in compiled app is not garbled. It's not easy to verify locally, because this project uses MFC and requires heavy VS workloads to build. It's best to only apply changes that are actually needed, i.e. fix typos, but preserve encoding as is.

Also, thinking about it more, why are these samples even placed here? They are regular C++ MFC samples and don't seem to have anything about .NET. And they are also not "snippets", they don't have snippet tags, but DO have project files and HTML readme, suggesting that they are actually intended to be complete samples.

Reply:
Should I get rid of all the NVC_MFC* solution folders?

Reply:
I get no hits searching for "NVC" except for one log file in one of the solution folders

Reply:
Yes, please remove them all. I didn't find any hits either.

Reply:
OK, I've removed them all, but I can split that out to a separate PR if preffered

Reply:
Oh my, that was a lot of files. Yes, a separate PR would be preferable. Thanks Nick!

Reply:
OK, I'll rebase out that commit to a new PR and remove the changes to those files in this one

Reply:
OK, separatated out the deletes, and rebased out all the MFC changes from this PR